0000-Proportional-Service-Provider Rewards

```diff
@@ -0,0 +1,63 @@
+# HIP XX: Proportional Service Provider Rewards
+- Author(s): @zer0tweets, @meowshka (Nova Labs, Inc.), @KeithRettig (Discord @KeithR)
+- Start Date: 2023-05-29
+- Category: Economic, Technical
+- Original HIP PR: #nnn
+- Tracking Issue: #nnn
+- Voting Requirements: veMOBILE Holders
+
+## Summary
+This HIP proposes to make modifications to how Service Providers are eligible for rewards from the Service Provider MOBILE Reward bucket.
+
+## Motivation
+The primary motivation is to prevent a scenario in which a Service Provider is rewarded the full 10% MOBILE reward bucket (provided HIP79 passes) while burning little to no Data Credits.
```

Comment:
Can you elaborate this section as well? This is a great place for context. Give a reader context about the mobile network, about the problems it is having, and the problem that sparked your desire to write this.

Side note: Your motivation is about gaming, right? I believe I've heard that as part of the working group's discussion. It would be good to add any info about how you all modeled gaming or expect gaming based on how gaming on the IOT network went to show how you identified the problem and how someone could game the network this way.